tools/downloader/pytorch_to_onnx.py

```diff
@@ -21,6 +21,14 @@ def positive_int_arg(values):
             sys.exit('Invalid value for input argument: {!r}, a positive integer is expected'.format(value))
     return result
 
+def model_parameters(parameters):
+    if not parameters:
+        return None
+    model_params = dict((param, value) for param, value in (element.split('=') for element in parameters.split(',')))
+    for param in model_params.keys():
+        model_params[param] = eval(model_params[param])
```

Comment:
This seems too ad-hoc to me. If you're going to use `eval` anyway, why not just take the entire constructor call as a script argument and `eval` the whole thing at once?

Reply:
I've already tried your suggestion, but got some problems. This is most suitable solution for this moment. I can seek the way to improve it, if you insist.

Reply:
I'd like to at least know what problems you ran into. Maybe we could come up with a better solution together.

> This is most suitable solution for this moment.

I should note that so far I don't even see the problem that this is a solution for. :-)